### PR TITLE
Roll back the lerna version change to app

### PR DESCRIPTION
**Pull request by benjdlambert**
Made a booboo yesterday and merged #4506 without my other commit.

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4545-zoxtuot8ho5pv5lwxding4qiloaxvifr.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602b9c2665f8171a50f15774

File: packages/create-app/templates/default-app/package.json.hbs [hunk 1]
```diff
@@ -30,7 +30,7 @@
   "devDependencies": {
     "@backstage/cli": "^{{version '@backstage/cli'}}",
     "@spotify/prettier-config": "^7.0.0",
-    "lerna": "^4.0.0",
+    "lerna": "^3.20.2",
```
Comment (Rugvip) [inline packages/create-app/templates/default-app/package.json.hbs:33]: But we wanna keep this yo

Comment (benjdlambert) [reply packages/create-app/templates/default-app/package.json.hbs:33]: why tho?

Comment (benjdlambert) [reply packages/create-app/templates/default-app/package.json.hbs:33]: oh shit - sorry just read the diff in reverse

Comment (benjdlambert) [reply packages/create-app/templates/default-app/package.json.hbs:33]: one sec

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4545-zoxtuot8ho5pv5lwxding4qiloaxvifr.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602b9c2665f8171a50f15774

File: .changeset/wicked-forks-sleep.md [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
+'@backstage/create-app': patch
 ---
 
-Upgrading to lerna@4.0.0. This changes the interface for importing, and because we can't run multiple versions of lerna, this is a breaking change.
-
-You'll need to update your root `package.json` like the following:
-
-```diff
--     "lerna": "^3.20.2",
-+     "lerna": "^4.0.0",
```
Comment (freben) [inline .changeset/wicked-forks-sleep.md:12]: do users not have to do this anymore?

**Review by freben — APPROVED** [review]

**Comment by benjdlambert** [comment]
Nope. It's not a required change. But maybe a future migration if we decide to use anything new that lerna might release

**Pull request merged**